src/components/DataInput/CheckboxGroupInput.tsx

```diff
@@ -3,7 +3,7 @@
 import React from "react";
 import { Checkbox, FormControl, FormControlLabel, FormGroup, FormLabel } from "@mui/material";
 
-interface Props {
+export interface Props {
```

Comment:
If props are being exported, rename them to `CheckboxGroupProps` or something more informative so users downstream don't get naming collisions.

Reply:
These interfaces are no longer being used and so I have removed the `export`s as well